lib/src/widgets/chat/typing/ds_dot_animation.widget.dart

```diff
@@ -0,0 +1,150 @@
+import 'package:flutter/material.dart';
+
+import 'ds_dot.widget.dart';
+
+/// Creates a list of animated dots for use in typing identification and other situations
+///
+/// To define the stitch appearance use [size] and [color]
+/// Use [numberOfDots] to set the amount of dots and use [innerPadding] to set the spacing between them
+/// To change the animation speed use [animationTime]
+/// To set the dot ascent level use [upLevelAnimation]
+class DSDotAnimation extends StatefulWidget {
+  final double size;
+  final Color color;
+  final int numberOfDots;
+  final double innerPadding;
+  final Duration animationTime;
+  final double upLevelAnimation;
+
+  const DSDotAnimation({
+    Key? key,
+    this.numberOfDots = 3,
+    this.size = 10,
```

Comment:
The default size should be 7px (or something like that) to match the prototype size.